utils/DeployUtils.ps1

```diff
@@ -181,31 +217,25 @@ function ConfigureScubaGearModule {
         [string]
         $PrereleaseTag = ""
     )
-    Write-Host "Configuring ScubaGear module..."
-    #TODO: Add any module configuration needed (e.g., adjust Module Version)
+    Write-Warning ">>> Configuring ScubaGear module..."
 
+    #TODO: Add any module configuration needed (e.g., adjust Module Version)
     # Verify that the module path folder exists
     if (Test-Path -Path $ModulePath) {
-        Write-Host "The module dir exists at "
-        Write-Host $ModulePath
+        Write-Warning ">>> The module dir exists at $ModulePath"
     }
     else {
-        Write-Warning "The module dir does not exist at "
-        Write-Warning $ModulePath
-        Write-Error "Failing..."
+        Write-Error ">>> Failed to find the module directory at $ModulePat."
```

Comment:
I think this is missing the h for ModulePath

Reply:
Fixed.